src/assets/locales/i18n.js

```diff
@@ -0,0 +1,15 @@
+// import de from './de/main.json';
+import en from './en/main';
```

Comment:
Maybe we should figure out how to handle i18n in our projects like this, jwplayer-fandom, tracking-opt-in and so on not to inject all translations into JS and load only needed ones?

Reply:
personally, I think all these as separated modules should include everything they need for itself and accept as fewer params as possible. Passing i18n from a client would require every app to maintain these translations, which might lead to mess.

Best way to achieve that would be to import JSONs dynamically, but with our current setup I'm afraid that additional code that would be needed to achieve that will be much larger than the actual size of 10 JSONs with 150 bytes each.

Reply:
Let's discuss it on our tech meet up next week